### PR TITLE
[AMI/GCP/Azure] - Align images name

### PR DESCRIPTION
**Pull request by yaronkaikov**
This PR contains the following commits:
[[AMI/GCP/Azure] - Align images name](https://github.com/scylladb/scylla-machine-image/pull/383/commits/3cd75beb69bd4c442ddf4bcb396b32c807e03976)

[Azure: sed base image type to Standard_D2_v4](https://github.com/scylladb/scylla-machine-image/pull/383/commits/3eb434ace9a41273b5a83c60d8f58a6fca2a4277)

[[cloud image tags] - Align tag names and values for all images](https://github.com/scylladb/scylla-machine-image/pull/383/commits/1bfe310bacdece0c1e84f0160a45e386f182d2a1)

[ami:add Name tag to match ami_name](https://github.com/scylladb/scylla-machine-image/pull/383/commits/1df4c29d032e52833f574bea1906e0cb9c7b9e9b)

### CODE REVIEW

**Comment by yaronkaikov** [comment]
Verification:
**AMI**: https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/ami/1488/
**GCE**: https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/gce-image/628/
**AZURE**: https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/azure-image/239/
**Next-machine-image**: https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/next-machine-image/383/ (Still running)

**Review by fruch — APPROVED** [review]
LGTM

File: packer/build_deb_image.sh [hunk 1]
```diff
@@ -320,7 +321,8 @@ set -x
   -var-file="$DIR"/variables.json \
   -var install_args="$INSTALL_ARGS" \
   -var ssh_username="$SSH_USERNAME" \
-  -var scylla_version="$SCYLLA_VERSION" \
+  -var scylla_full_version="$SCYLLA_FULL_VERSION" \
+  -var version="$VERSION" \
```
Comment (benipeled) [inline packer/build_deb_image.sh:325]: /version/scylla_version/ - not critical but to make it clear like the other tools on the next rows

File: packer/scylla.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
           "virtual_name": "ephemeral7"
         }
       ],
-      "ami_name": "{{user `image_prefix`}}scylla_{{isotime | clean_resource_name}}-build-{{user `scylla_build_id`| clean_resource_name}}-{{user `arch`| clean_resource_name}}",
+      "ami_name": "{{user `image_prefix`}}{{user `product` | clean_resource_name}}-{{user `version` | clean_resource_name}}-{{user `arch`| clean_resource_name}}-{{isotime | clean_resource_name}}",
```
Comment (benipeled) [inline packer/scylla.json:41]: We can avoid duplicating it for each builder by a var that will be used by all of them

File: packer/build_deb_image.sh (hunk 1 above)
Comment (yaronkaikov) [reply packer/build_deb_image.sh:325]: I missed this comment , will send a follow up PR

Comment (yaronkaikov) [reply packer/build_deb_image.sh:325]: https://github.com/scylladb/scylla-machine-image/pull/384

File: packer/scylla.json (hunk 2 above)
Comment (yaronkaikov) [reply packer/scylla.json:41]: Tried to implement it but it's not working (probably i did something wrong). Opened an issue : https://github.com/scylladb/scylla-machine-image/issues/385

**Pull request merged**